app/views/prayers/edit.html.erb

```diff
@@ -4,8 +4,15 @@
     <%= form_for @prayer, html:{class: 'form-mobile-width'} do |f|%>
       <div class="form-group">
         <%= f.label 'Prayer Request: ' %>
+        <div id="change_status_prompt"></div>
         <%= f.text_area :request, class: 'form-control', rows: "5", required: true %>
       </div>
+      <div class="form-group">
+        <%= f.label 'Status: '%>
+        <%= f.select :status, Prayer.statuses.keys.to_a %>
+        <%= f.hidden_field :status, :id => 'curr_status' %>
```

Comment:
@TimothyHou my prayer status is not updated with the new status. The POST request pass in the old status instead?